### PR TITLE
INT-24 updated JobVacancies component to get vacancy data from heroku…

### PR DESCRIPTION
**Pull request by PH86**
… server

### CODE REVIEW

File: src/pages/JobVacancies/JobVacancies.tsx [hunk 1]
```diff
@@ -1,181 +1,190 @@
...
+    const { openModal, setOpenModal, showVacancy } = React.useContext(AppContext);
+    const [search, setSearch] = React.useState<string>('');
+    const [searchFilter, setSearchFilter] = React.useState<string>('title');
+    const [vacancies, setVacancies] = React.useState<IJobData[]>([]);
 
-	const toggleModal = () => {
-		setOpenModal(!openModal)
-	}
+    React.useEffect(() => {
+        fetch('http://interview-server-heroku.herokuapp.com/vacancies', {headers: {'Content-Type': 'application/json'}})
```
Comment (sambuckley1995) [inline src/pages/JobVacancies/JobVacancies.tsx]: Probably better to pop this into .env

**Review by sambuckley1995 — CHANGES_REQUESTED** [review]

File: src/pages/JobVacancies/JobVacancies.tsx [hunk 2]
```diff
@@ -1,181 +1,190 @@
...
+            .then((response) => response.json())
```
Comment (JackHewitt08) [inline src/pages/JobVacancies/JobVacancies.tsx]: I think we could combine these promises here.

.then((response) => {
    response.json()
    setVacancies
}

**Review by JackHewitt08 — APPROVED** [review]
Approved but please complete the comment

File: src/pages/JobVacancies/JobVacancies.tsx (hunk 1 above)
Comment (PH86) [reply src/pages/JobVacancies/JobVacancies.tsx]: Changed on new pull request

**Review by sambuckley1995 — APPROVED** [review]

**Pull request merged**